internal/controllers/topology/cluster/reconcile_state_test.go

```diff
@@ -1350,6 +1360,7 @@ func TestReconcileReferencedObjectSequences(t *testing.T) {
 									"controllerManager": map[string]interface{}{
 										"extraArgs": map[string]interface{}{
 											"enable-hostpath-provisioner": "true",
```

Comment:
Are all the comments in this test case intended?

Reply:
I have to take another look at this test with @chrischdi

Reply:
Was fixed in https://github.com/kubernetes-sigs/cluster-api/pull/6495/commits/c07bca1c3aea0366c654ca418bbc42b72dd1f595